napari_allencell_annotator/widgets/ome_zarr_directory_or_file_dialog.py

```diff
@@ -0,0 +1,44 @@
+from PyQt5.QtWidgets import QListView, QAbstractItemView, QTreeView
+from qtpy.QtWidgets import QFileDialog
+from napari_allencell_annotator._style import Style
+
+
+class OmeZarrDirectoryOrFileDialog(QFileDialog):
+    """
+    A custom QFileDialog that allows the user to select files and zarr directories.
+    """
+
+    def __init__(self, parent, title):
+        super().__init__(parent, title)
+        self.setStyleSheet(Style.get_stylesheet("main.qss"))
+        self.currentChanged.connect(self._selected)
+        self.setNameFilter("Directories and files (*)")
+        self.setOption(QFileDialog.DontUseNativeDialog, True)
+
+    def _selected(self, name: str) -> None:
+        """
+        Called whenever the user selects a new option in the File Dialog menu.
+        """
+        self.setFileMode(QFileDialog.Directory | QFileDialog.ExistingFiles)
+        self.setNameFilter("Directories and files (*)")
+        self.setOption(QFileDialog.DontUseNativeDialog, True)
+        self.findChild(QListView, "listView").setSelectionMode(QAbstractItemView.ExtendedSelection)
+
+        f_tree_view: QTreeView = self.findChild(QTreeView)
+        if f_tree_view:
+            f_tree_view.setSelectionMode(QAbstractItemView.ExtendedSelection)
+
+    def accept(self):
```

Comment:
return type

Reply:
added in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/5d6c4d38d51fab0f3a1af0cbdb96244fa25b0cf6